src/server.cpp

```diff
@@ -1800,6 +1800,15 @@ void Server::SendSetSky(u16 peer_id, const video::SColor &bgcolor,
 	Send(&pkt);
 }
 
+void Server::SendSetClouds(u16 peer_id, const float density,
+		const video::SColor &color, const float glow)
+{
+	NetworkPacket pkt(TOCLIENT_SET_CLOUDS, 0, peer_id);
```

Comment:
because size could be known before inserting datas, please set the default size to 2 + sizeof(color) + 2 (staticly)